xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -82,8 +84,23 @@ namespace PVR
     std::string   m_strFanartPath;    /*!< fanart path */
     unsigned      m_iRecordingId;     /*!< id that won't change while xbmc is running */
 
+  private:
     CPVRRecording(void);
     CPVRRecording(const PVR_RECORDING &recording, unsigned int iClientId);
+
+    // Prevent copy construction, even for CPVRRecording instances and friends.
+    // Note: Only declared, but intentionally not implemented
+    //       to prevent compiler generated copy ctor and to force
+    //       a linker error in case somebody tries to call it.
+    CPVRRecording(const CPVRRecording &tag);
```

Comment:
I red what @davilla wrote. It was just a short comment re "friend is evil", not "unimplemented cp-ctor and op=".

I'm pretty sure that all linkers on all relevant platforms will behave like expected, because the "Voodoo" stuff we're talking about is common practice since the ninetees. Looking up symbols and exiting with an error if not found is absolutely Basic linker functionality, nothing special at all.

I will keep this at it is now. If it bombs on any platform we will detect this early (e.g. Jenkins before merge) and I promise to implement those two member functions lightning fast. ;-)